src/arena/arena_loader.py

```diff
@@ -46,7 +46,10 @@ def load_arena_by_name(name: str) -> Arena:
                 "initial_spawn_for_player"
             )
             initial_spawn_for_player = None
-            if initial_spawn_for_player_str is not None:
+            if (
+                initial_spawn_for_player_str is not None
+                and initial_spawn_for_player_str != ""
+            ):
```

Comment:
Unrelated QOL improvement: empty string will be treated the same as if the attribute is unset.